lib/Controller/MailboxesController.php

```diff
@@ -246,10 +246,10 @@ public function update() {
 	 * @throws ClientException
 	 */
 	#[TrapError]
-	public function create(int $accountId, string $name): JSONResponse {
+	public function create(int $accountId, string $name, bool $isSentMailbox = false): JSONResponse {
```

Comment:
This makes the new parameter only usable for specialuse=sent. Change it to allow a generic specialuse attribute. Then we will be able to also create other special mailboxes.